app/src/main/res/drawable/button_style_border.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <solid android:color="@color/colorBackground" />
+
```

Comment:
formatting required in these drawable resource files also

Reply:
What kind of formatting, can you be more specific sir?